tests/integration_tests/functional/test_snapshot_basic.py

```diff
@@ -352,6 +353,106 @@ def test_5_inc_snapshots(network_config,
     test_matrix.run_test(_test_seq_snapshots)
 
 
+DEFAULT_HOST_IP = "192.168.0.1"
+DEFAULT_GUEST_IP = "192.168.0.2"
+DEFAULT_TAP_NAME = "tap0"
+DEFAULT_DEV_NAME = "eth0"
+DEFAULT_NETMASK = 30
+
+
+def _get_process_start_time(p_pid):
+    # Helper function - returns the start_time and returncode for
```

Comment:
This comment is misleading:

return code provided by this function is not `return code of provided PID`.

It is actually `return code of 'stat -c%X <pid>'`

Reply:
Done